language/transaction-builder-generated/Cargo.toml

```diff
@@ -1,15 +1,17 @@
 [package]
-name = "libra-transaction-scripts"
+name = "transaction-builder-generated"
 version = "0.1.0"
-edition = "2018"
 authors = ["Libra Association <opensource@libra.org>"]
-description = "Libra transaction scripts"
+description = "Libra transaction-builder"
```

Comment:
"Generated transaction builders"?

Actually it would be nice to have `stdlib` in the Rust namespace to be future-proof. (In the future, there could be several such libraries for different bundles of scripts.)

Reply:
To be concrete, `language/transaction-builder/generated/src/stdlib.rs` could define the module `stdlib` of crate `transaction-builder-generated`, maybe?